src/sources/soundsourceffmpeg.h

```diff
@@ -88,6 +88,11 @@ class SoundSourceProviderFFmpeg: public SoundSourceProvider {
         return "FFmpeg";
     }
 
+    SoundSourceProviderPriority getPriorityHint() const override {
+        // FFmpeg will only be used as the last resort
+        return SoundSourceProviderPriority::LOWEST;
```

Comment:
We could add a parameter with the file extension as you mentioned below. This will be sufficient until Mixxx determines the file type by the actual content instead of only using the file extension.